README.md

```diff
@@ -1,38 +1,56 @@
-# Cloud Native Networking Preamble
+# Cloud Native Principles
 
-**What is a Cloud Native Network Function \(CNF\)?**
+To talk about cloud native applications and systems, we need to define cloud native.  CNCF provides a high-level overview of what it means to be cloud native and lists some implementations and the benefits they provide in the [CNCF Cloud Native Definition v1.0 defintion](https://github.com/cncf/toc/blob/master/DEFINITION.md).
```

Comment:
"CNCF Cloud Native Definition v1.0 defintion"
> should be CNCF Cloud Native Definition v1.0

Reply:
Are we allowed to critique their defintion?  As defintions go it actually doesn't defint mcuh.

Reply:
@iawells the [README](https://github.com/cloud-native-principles/cloud-native-principles/blob/b7013009874e0c8f043781c6e2cd54e59c9b763d/README.md) somewhat addresses this.  Including 

> CNCF provides a high-level overview of what it means to be cloud native and lists some implementations and the benefits they provide in the CNCF Cloud Native Definition v1.0.

The benefits and implementation items are mixed into the v1.0 "definition".  We attempted to look at the underlying principles for implementation techniques like containers and service meshes which we then list int the [fundamental concepts](https://github.com/cloud-native-principles/cloud-native-principles/blob/b7013009874e0c8f043781c6e2cd54e59c9b763d/README.md#principles) list.

Reply:
I have gone back and forth on this one. I think diplomacy is the path here, expounding upon what they have and providing context to the fluffier parts. I think if we wanted to critique their definition directly, the repo that hosts it would be the best place for that.